assets/scss/base/_lists.scss

```diff
@@ -62,6 +62,89 @@ ul, nav ul {
   li {
     display: inline;
   }
+
+  .list__item + .list__item {
+    margin-left: em(15);
+  }
+}
+
+
+/*
+.list--collapse
+
+A collapsable list element that displays inline on larger (Desktop) breakpoints and block with a toggle to collapse/exapand the list children. All of the list items have the 'list__item' class and the toggle item has the attribute 'data-list--collapse-toggle'
+
+Markup:
+<ul class="list--collapse hide-inline-list--collapse-targets">
+    <li class="list__item" data-list--collapse-toggle>
+        <span>I'm the toggle</span>
+    </li>
+    <li class="list__item">
+        <a href="#">list item link</a>
+    </li>
+    <li class="list__item">
+        <a href="#">another list item link</a>
+    </li>
+</ul>
+
+Styleguide Lists.Collapse
+*/
+
+.list--collapse {
+
+  li {
```

Comment:
I could replace:

```scss
.list--collapse {
  li {}
}
```

with:
```scss
.list__item {
  
}
```

I wanted to avoid chances of polluting any possible future use of the .list__item class elsewhere